plugins/sharding/logical_range_filter.rb

```diff
@@ -970,10 +972,14 @@ def filter_by_range(range_index, min, min_border, max, max_border)
                     n_matched_records = index_cursor.select(result_set, options)
                   end
                 end
+                query_logger.log(:size, ":",
+                                 "filter(#{n_matched_records}): #{@shard.table_name}.#{@filter}")
```

Comment:
`filter(N)[TABLE_NAME]: FILTER` is better.

Reply:
I'll fix it.

Reply:
Really? Is `"filter(#{n_matched_records}): #{@shard.table_name}.#{@filter}"` expected?

Reply:
Oops!